aviary/subsystems/aerodynamics/flops_based/solved_alpha_group.py

```diff
@@ -33,7 +33,7 @@ def initialize(self):
                                   'drag coefficient table as a function of altitude, '
                                   'Mach, and angle of attack')
 
-        self.options.declare('training_data', default=False,
+        self.options.declare('is_training_data', default=False,
```

Comment:
Maybe "input_training_data" would be a better name for this.

Reply:
I will use `connect_training_data` for now.